utils/management_tools/Get_runs_with_most_fields.ipynb

```diff
@@ -59,7 +59,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Enter project to scan"
+    "## Enter project to scan\n",
+    "\n",
+    "To find the full project name:\n",
+    "\n",
+    "1. [Log in to Neptune](https://app.neptune.ai/).\n",
+    "1. The workspace name is displayed in the top-left corner of the app. To copy the project path, in the top-right corner, open the settings menu and select **Properties**.\n",
```

Comment:
I need to update the Notebook template :')

```suggestion
    "1. Open the project settings and select **Edit project details**.\n",
```

Reply:
We'll need to update all notebooks after the Apri release :')